src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -75,12 +82,36 @@
   background-color: var(--pf-c-dropdown__toggle--BackgroundColor);
   border: none;
 
+  &.pf-m-split-button {
+    padding: 0;
+
+    > * {
+      padding-top: var(--pf-c-dropdown__toggle--m-split-button--child--PaddingTop);
+      padding-right: var(--pf-c-dropdown__toggle--m-split-button--child--PaddingRight);
+      padding-bottom: var(--pf-c-dropdown__toggle--m-split-button--child--PaddingBottom);
+      padding-left: var(--pf-c-dropdown__toggle--m-split-button--child--PaddingBottom);
```

Comment:
I think this should be `padding-left: var(--pf-c-dropdown__toggle--m-split-button--child--PaddingLeft);`

Reply:
Nice, this section was all messed up :)